rllib/env/single_agent_env_runner.py

```diff
@@ -393,6 +393,11 @@ def _sample(
                         observation_space=self.env.single_observation_space,
                         action_space=self.env.single_action_space,
                     )
+                    self._make_on_episode_callback(
```

Comment:
Nice, so this works again.

Reply:
Yeah, b/c the new gym 1.x does NOT automatically reset a done vector subenv anymore, but requires you to take the extra step from terminal observation to reset observation "manually". This allows us to slip in the callback again.